### ISSUE TITLE
Content type issue

### ISSUE THREAD

**Issue by phones24** [nikkiii/embedhttp#8]
In case of using StaticFileHandler setting content type to "text/html" for every file prevents browser from recognizing its content (i.e. for css)
 
if (!resp.getHeaders().containsKey(HttpHeader.CONTENT_TYPE)) {
	resp.addHeader(HttpHeader.CONTENT_TYPE, "text/html");
}

**Comment by mahmed8003** [comment]
I updated the file like this, and now it working fine.
```java
/**
 * A RequestHandler to handle serving of simple static files
 * 
 * @author Nikki
 * 
 */
public class HttpStaticFileHandler implements HttpRequestHandler {

	private static final String CONTENT_TYPE = "Content-Type";

	/**
	 * The document root
	 */
	private File documentRoot;

	/**
	 * The document root's path, cached to save time.
	 */
	private String documentRootPath;

	/**
	 * Construct a new static file server
	 * 
	 * @param documentRoot
	 *            The document root
	 */
	public HttpStaticFileHandler(File documentRoot) {
		this.documentRoot = documentRoot;
	}

	@Override
	public HttpResponse handleRequest(HttpRequest request) {
		String uri = request.getUri();
		try {
			uri = URLDecoder.decode(uri, "UTF-8");
		} catch (UnsupportedEncodingException e1) {
			uri = uri.replace("%20", " ");
		}
		File file = new File(documentRoot, uri);
		if (file.exists() && !file.isDirectory()) {
			try {
				if (documentRootPath == null) {
					documentRootPath = documentRoot.getAbsolutePath();
					if (documentRootPath.endsWith("/") || documentRootPath.endsWith(".")) {
						documentRootPath = documentRootPath.substring(0, documentRootPath.length() - 1);
					}
				}
				String requestPath = file.getCanonicalPath();
				if (requestPath.endsWith("/")) {
					requestPath = requestPath.substring(0, requestPath.length() - 1);
				}
				if (!requestPath.startsWith(documentRootPath)) {
					return new HttpResponse(HttpStatus.BAD_REQUEST, HttpStatus.BAD_REQUEST.toString());
				}
			} catch (IOException e) {
				return new HttpResponse(HttpStatus.INTERNAL_SERVER_ERROR, HttpStatus.INTERNAL_SERVER_ERROR.toString());
			}
			try {
				HttpResponse res = new HttpResponse(HttpStatus.OK, new FileInputStream(file));
				String ext = getExtensionOfFile(file);
				if(ext.equalsIgnoreCase("png")) {
					res.addHeader(CONTENT_TYPE, "image/png");

				}else if(ext.equalsIgnoreCase("jpg") || ext.equalsIgnoreCase("jpeg")) {
					res.addHeader(CONTENT_TYPE, "image/jpeg");

				}else if(ext.equalsIgnoreCase("json")) {
					res.addHeader(CONTENT_TYPE, "application/json");

				}else if(ext.equalsIgnoreCase("xml")) {
					res.addHeader(CONTENT_TYPE, "application/xml");

				}else if(ext.equalsIgnoreCase("txt")) {
					res.addHeader(CONTENT_TYPE, "text/plain");
				} else {
					res.addHeader(CONTENT_TYPE, "text/html");
				}
				res.setResponseLength(file.length());
				return res;
			} catch (IOException e) {
				e.printStackTrace();
			}
		}
		return null;
	}

	private String getExtensionOfFile(File file) {
		String extension = "";
		String fileName = file.getName();
		int i = fileName.lastIndexOf('.');
		if (i > 0) {
			extension = fileName.substring(i+1);
		}
		return extension;
	}

}
```

**Comment by nikkiii** [comment]
Sorry for the incredibly late response, this was fixed in 1.1.1 and includes a full content type map.
